core/src/main/java/hivemall/mf/FactorizedModel.java

```diff
@@ -30,25 +30,25 @@
 import javax.annotation.concurrent.NotThreadSafe;
 
 @NotThreadSafe
-public final class FactorizedModel {
+public class FactorizedModel {
```

Comment:
It seems FactorizedModel is not used in Cofactor. 

Is this change required? Revert if not used.